libkbfs/tlf_edit_history.go

```diff
@@ -339,6 +339,11 @@ func (teh *TlfEditHistory) setEdits(ctx context.Context,
 		if len(list) > desiredEditsPerWriter {
 			list = list[len(list)-desiredEditsPerWriter:]
 		}
+		// Don't copy the ops, we don't need them anymore and we don't
+		// want shallow copies of them getting out.
+		for i := range list {
+			list[i].op = nil
+		}
 		currEdits[w] = list
```

Comment:
It's the value we're iterating over though. Didn't realize you could mutate your iterable in place.